x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/esnative/NativeUsersStore.java

```diff
@@ -284,9 +290,10 @@ public void updateReservedUser(
         char[] passwordHash,
         DocWriteRequest.OpType opType,
         RefreshPolicy refresh,
+        Consumer<Exception> consumer,
```

Comment:
Do we need this ? We only ever pass the onFailure of the ActionListener we also pass as a parameter, so we can just simply leave this as it was ?

Reply:
Happy to push this change if needed, no worries

Reply:
Please do! I merged the conflict, but there are some failures I don't have time to look at (my flight is in 2hr ))).

Reply:
> Do we need this ? We only ever pass the onFailure of the ActionListener we also pass as a parameter, so we can just simply leave this as it was ?

I use it for the special error handling to return INTERNAL_SERVER_ERROR, look at `storeAutoconfiguredElasticUser`